esi_leap/tests/db/sqlalchemy/test_api.py

```diff
@@ -259,4 +266,122 @@ def test_offer_get_by_name(self):
 
     def test_offer_get_by_name_not_found(self):
 
-        assert api.offer_get_by_uuid('some_uuid') is None
+        self.assertEqual(api.offer_get_by_name('some_name'), [])
+
+    def test_offer_destroy(self):
+
```

Comment:
You don't actually need this blank line after each function definition (I think you probably saw one early on and thought it was the standard).

Reply:
I deleted all the blank lines after the functions defs.